### ISSUE TITLE
Wrong version compared

### ISSUE THREAD

**Issue by viretp** [jagodevreede/semver-check#66]
It seems that the plugin does the check in the wrong direction:

- if I add a new public class Toto, I get following message:
```
[INFO] --- semver-check:0.4.1:check (check) @ xxx-common ---
...
[INFO] Class xxx.Toto is removed
[INFO] Determined SemVer type as major and is currently patch, next version should be: 101.0.0

```
I would expect a minor update when adding a public class.

- after removyl of 2 public classes, I get following message:
```
[INFO] --- semver-check:0.4.1:check (check) @ xxx-common ---
...
[INFO] Checking SemVer against last known version 100.0.0
[WARNING] Failed to load class module-info from xxx-common-100.0.1-viretp-SNAPSHOT.jar due to: java.lang.NoClassDefFoundError module-info is not a class because access_flag ACC_MODULE is set
[WARNING] Failed to load class module-info from xxx-common-100.0.0.jar due to: java.lang.NoClassDefFoundError module-info is not a class because access_flag ACC_MODULE is set
[INFO] File META-INF/MANIFEST.MF has been changed
[INFO] Determined SemVer type as patch and is currently patch, next version should be: 100.0.1

```
I would expect a major update when removing public classes.

Maybe the problem has to do with the module-info file (see WARNING) ?

**Comment by jagodevreede** [comment]
I've fixed some bugs in version 0.5.0 regarding non-public classes, and removal of classes/packages. Also, 0.5.0 will no longer have the `Failed to load class module-info`.

Please test version 0.5.0 if the problem still exists.

**Comment by jagodevreede** [comment]
This issue is fixed in [version 0.5.1](https://github.com/jagodevreede/semver-check/releases/tag/v0.5.1), the jar files were swapped in the Maven plugin.
